mindsdb/api/mysql/mysql_proxy/classes/sql_query.py

```diff
@@ -65,6 +65,12 @@ def parse_insert(sql):
         values = p['literal'] if isinstance(p, dict) else p
         if isinstance(values, list) is False:
             values = [values]
+        for i, v in enumerate(values):
+            if isinstance(v, dict) and 'literal' in v:
```

Comment:
Is `isinstance(v, dict) and 'literal' not in v` case supposed to be ignored?
Making it explicit will remove such doubts for anyone reading the code:
```
for i, v in enumerate(values):
            if isinstance(v, dict):
                if 'literal' in v:
                     v = v['literal']
                else:
                    pass
```

Reply:
yeah, may be it will be easier to read